skipruntime-ts/examples/departures.ts

```diff
@@ -58,7 +58,8 @@ const service = await runService(
         departuresFromAPI: new Polled(
           "https://api.unhcr.org/rsq/v1/departures",
           10000,
-          (data: Result) => data.results.map((v, idx) => [idx, [v]]),
+          (data: Result) =>
+            data.results.map((v, idx) => [JSON.stringify(idx), [v]]),
```

Comment:
`idx` here is guaranteed to be a `number`.
```suggestion
            data.results.map((v, idx) => [idx.toString(), [v]]),
```